src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -555,18 +556,16 @@ export class InputTimePicker
   };
 
   private async loadDateTimeLocaleData(): Promise<void> {
-    const normalizedLocale = this.getNormalizedLocale();
+    const supportedLocale = getSupportedLocale(this.effectiveLocale).toLowerCase();
 
-    if (normalizedLocale === "en" || normalizedLocale === "en-us") {
-      return;
-    }
-
-    const { default: localeConfig } = await supportedDayJsLocaleToLocaleConfigImport.get(
-      normalizedLocale
-    )();
+    if (supportedDayJsLocaleToLocaleConfigImport.has(supportedLocale)) {
```

Comment:
You still need the previous mappings you had for `pt-PT` and `no`. Once this is in place, you no longer need to check if the locale is in the import map since all supported locales are mapped to a corresponding locale config.

Reply:
good catch, updated!